apps/vc-api/src/vc-api/vc-api.controller.ts

```diff
@@ -51,6 +53,21 @@ export class VcApiController {
     return await this.vcApiService.issueCredential(issueDto);
   }
 
+  /**
+   * Verify a credential. Conforms to https://w3c-ccg.github.io/vc-api/#verify-credential
+   * @returns verification results: checks, warnings, errors
+   */
+  @Post('/credentials/verify')
+  @HttpCode(200)
+  @ApiResponse({ status: 200, description: 'Verifiable Credential successfully verified' })
```

Comment:
These strings come from w3c, but not sure if we need to implement them verbatim https://w3c-ccg.github.io/vc-api/#verify-credential